pkg/util/ec2/ec2.go

```diff
@@ -35,6 +36,34 @@ func GetHostname() (string, error) {
 	return getMetadataItemWithMaxLength("/hostname", config.Datadog.GetInt("metadata_endpoints_max_hostname_size"))
 }
 
+func GetNetworkID() (string, error) {
+	resp, err := getMetadataItem("/network/interfaces/macs")
+	if err != nil {
+		return "", err
+	}
+
+	macs := strings.Split(strings.TrimSpace(resp), "\n")
+	vpcIDs := common.NewStringSet()
```

Comment:
Not familiar with GCE and AWS networking: is having the same network ID for multiple `mac` a real use case ?